instrumentation/elasticsearch/elasticsearch-transport-5.3/src/test/groovy/springdata/Elasticsearch53SpringRepositoryTest.groovy

```diff
@@ -30,317 +30,317 @@ import static io.opentelemetry.trace.Span.Kind.CLIENT
 import static io.opentelemetry.trace.Span.Kind.INTERNAL
 
 class Elasticsearch53SpringRepositoryTest extends AgentTestRunner {
-  // Setting up appContext & repo with @Shared doesn't allow
-  // spring-data instrumentation to applied.
-  // To change the timing without adding ugly checks everywhere -
-  // use a dynamic proxy.  There's probably a more "groovy" way to do this.
+    // Setting up appContext & repo with @Shared doesn't allow
```

Comment:
can you try reformatting this file in Intellij? i'm not sure why it's not using our `.editorconfig`